config/deploy/data-rearchitecture-project-test.rb

```diff
@@ -0,0 +1,11 @@
+set :branch, 'data-rearchitecture-for-dashboard'
+set :rails_env, 'development'
```

Comment:
I'm not sure if 'development' rail environment is the best choice here

Reply:
No. I suggest you use 'production'.